plugins/BulkTracking/Tracker/Handler.php

```diff
@@ -40,18 +44,22 @@ public function onAllRequestsTracked(Tracker $tracker, RequestSet $requestSet)
 
     public function process(Tracker $tracker, RequestSet $requestSet)
     {
-        $invalidRequests = 0;
-        foreach ($requestSet->getRequests() as $request) {
+        $isAuthenticated = $this->isBulkTrackingRequestAuthenticated($requestSet);
+
+        /** @var Response $response */
+        $response = $this->getResponse();
+        $response->setIsAuthenticated($isAuthenticated);
```

Comment:
Is this done here and not in `Response` because you cannot access the `RequestSet` in `Response`?